android/sound.cpp

```diff
@@ -249,17 +249,10 @@ oboe::DataCallbackResult CSound::onAudioInput ( oboe::AudioStream* oboeStream, v
     // We're good to start recording now
     // Take the data from the recording device output buffer and move
     // it to the vector ready to send up to the server
-    float* floatData = static_cast<float*> ( audioData );
+    int16_t* floatData = static_cast<int16_t*> ( audioData );
 
     // Copy recording data to internal vector
-    for ( int frmNum = 0; frmNum < numFrames; ++frmNum )
-    {
-        for ( int channelNum = 0; channelNum < oboeStream->getChannelCount(); channelNum++ )
-        {
-            vecsTmpInputAudioSndCrdStereo[frmNum * oboeStream->getChannelCount() + channelNum] =
-                static_cast<int16_t>(floatData[frmNum * oboeStream->getChannelCount() + channelNum] * _MAXSHORT);
-        }
-    }
+    memcpy(vecsTmpInputAudioSndCrdStereo.data(),floatData,sizeof(int16_t)*numFrames * oboeStream->getChannelCount());
```

Comment:
I think the style is not correct here.

Reply:
Thanks, is there a .clang-format I can use? I believe I saw something in the issues/discussions but I don't know if it finally materialized.

Reply:
Yes, it's here: https://github.com/jamulussoftware/jamulus/pull/1127/files#diff-1026e0038b722990204a42bed8a6f7c0ec2302aa79e3fad1959d62ba968edfa2

It's still not merged and done for existing code yet, but new code can certainly be run through this.

Reply:
ok, new commit after applying .clang-format as indicated by @hoffie 
Thanks